ac

```diff
@@ -265,6 +265,21 @@ ac_install(){
     fi
 }
 
+# ------------------------------------------------------------------------------
+# Run ansible-lint on the locally checked out GH Branch
+# ------------------------------------------------------------------------------
+#->ac-lint:
+## Run ansible-lint on the local GH branch.
+## Usage: ac [-b, --ac-build]
+## Example:
+##  $ ac --ac-build
+/Users/ddimatos/git/github/ibm_zos_core/venv/venv-2.15/bin/
+ac_ansible_lint(){
+    gh_branch=`git branch |grep "*" | cut -d" " -f2`
+    message "Linting with ansible-lint on GH branch: '$gh_branch'."
+    . $VENV_BIN/activate && $VENV_BIN/ansible-lint
```

Comment:
Should the second call also have '.' (source) at the beginning, since it's a separate function?  I'm not sure here, because I haven't done much with virtual environments.

Reply:
Hey @richp405 , you don't need a second sourcing `.` once the venv has started, you can resolve this after reading.